android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/NearbyBeaconsFragment.java

```diff
@@ -398,15 +427,20 @@ private void setText(View view, int textViewId, String text) {
     @SuppressLint("InflateParams")
     @Override
     public View getView(int i, View view, ViewGroup viewGroup) {
-      // Get the list view item for the given position
-      if (view == null) {
-        view = getActivity().getLayoutInflater().inflate(R.layout.list_item_nearby_beacon,
-                                                         viewGroup, false);
-      }
-
       // Display the pwsResult.
       PwPair pwPair = getItem(i);
       PwsResult pwsResult = pwPair.getPwsResult();
+      if (isFolderItem(pwPair)) {
+        view = getActivity().getLayoutInflater().inflate(R.layout.folder_item_nearby_beacon,
+            viewGroup, false);
+        WifiManager wifiManager = (WifiManager) getActivity().
+            getSystemService(Context.WIFI_SERVICE);
+        WifiInfo wifiInfo = wifiManager.getConnectionInfo();
+        setText(view, R.id.title, wifiInfo.getSSID().trim().replace("\"", ""));
```

Comment:
It would be better to only strip off the leading and trailing quotes rather than removing all quotes since there might be " characters in the SSID itself.